docs/source/installation.rst

```diff
@@ -0,0 +1,52 @@
+Requirements
+============
+
+You need Python 3.7 or later to run GalaxyChop.
+
+
+Installation
+============
+
+
+This is the recommended way to install GalaxyChop.
+
+Installing  with pip
+^^^^^^^^^^^^^^^^^^^^
+
+Make sure that the Python interpreter can load GalaxyChop code.
+The most convenient way to do this is to use virtualenv, virtualenvwrapper, and pip.
+
+After setting up and activating the virtualenv, run the following command:
+
+.. code-block:: console
+
+   $ pip install GalaxyChop
+
+That should be it all.
+
+
+
+Installing the development version
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you’d like to be able to update your GalaxyChop code occasionally with the
+latest bug fixes and improvements, follow these instructions:
```

Comment:
Acá le cambiaria el orden "improvements and bug fixes", sino queda como que siempre tiene bugs xD